safe/gui/tools/shake_grid/test/test_shake_grid.py

```diff
@@ -62,62 +81,75 @@ def check_feature_count(path, count):
             message = 'Expected %s features, got %s' % (count, feature_count)
         return flag, message
 
+    def test_conversion(self):
+        """Test conversion for getting the raster."""
+        normal_shakemap_tif = NORMAL_SHAKE_GRID.mmi_to_raster()
+        normal_contour_shp = NORMAL_SHAKE_GRID.mmi_to_contours()
+
+        smoothed_shakemap_tif = SMOOTHED_SHAKE_GRID.mmi_to_raster()
+        smoothed_contour_shp = SMOOTHED_SHAKE_GRID.mmi_to_contours()
+
+        print normal_shakemap_tif
+        print normal_contour_shp
+        print smoothed_shakemap_tif
+        print smoothed_contour_shp
```

Comment:
We can move to python 3: `print()` it works on python two too ;-)

Reply:
Removed.